scripts/build/gen_isr_tables.py

```diff
@@ -14,6 +14,9 @@
 from elftools.elf.sections import SymbolTableSection
 
 ISR_FLAG_DIRECT = (1 << 0)
+ISR_FLAG_SHARE  = (1 << 1)
```

Comment:
Unless there's a significant runtime savings for "unshared" IRQs that demands this flag, wouldn't it be nicer for the API to just work?  Register more than one ISR on the same interrupt and they all get called, no muss, no fuss.

Also avoids the significant[1] danger involved in duplicating a C API between headers and python code.

[1] Really.  I've absolutely tripped over issues like this.  It's a pretty bad footgun, because you really don't expect to need to look in python when you change a preprocessor symbol!

Reply:
I agree. I've removed the flags and any number of shared interrupts can be generated with IRQ_DIRECT_CONNECT